src/renderers/webgl/WebGLProgram.js

```diff
@@ -689,6 +691,7 @@ function WebGLProgram( renderer, cacheKey, parameters, bindingStates ) {
 		versionString = '#version 300 es\n';
 
 		prefixVertex = [
+			'precision highp sampler2DArray;',
```

Comment:
@bhouston I've seen other code (e.g. https://forum.unity.com/threads/texture2darray-broken.1109753/) where `mediump` is used instead of `highp` in context of texture arrays. Do you think `mediump` is sufficient for our use case?

Reply:
Maybe we can try mediump until someone complains? 🤓